README.rst

```diff
@@ -119,15 +119,17 @@ Generate Terraform variables:
    controller_disk_size = 0
 
    ssh_public_key = "~/.ssh/changeme.pub"
+   ssh_user       = "cloud-user"
 
    storage_count  = "3"
    storage_flavor = "general.v1.small"
    storage_disk_size = 100
 
    EOF
 
-You will need to set the `multinode_flavor`, `multinode_keypair`, `prefix`, and
-`ssh_public_key`.
+You will need to set the `multinode_flavor`, `multinode_keypair`, `prefix`,
+`ssh_public_key`, and optionally `ssh_user` if you are using a different
+distribution.
```

Comment:
'a different distribution' might be confusing for someone using a multinode for the first time. Probably safest just to say something like 'a distribution that does not have a `cloud-user` account'

Reply:
Resolved